packages/web-runtime/src/components/Topbar/UserMenu.vue

```diff
@@ -58,12 +58,15 @@
           <oc-icon name="cloud" fill-type="line" class="oc-p-xs" />
           <div class="storage-wrapper-text">
             <p class="oc-my-rm">
-              <span v-text="personalStorageLabel" />
-              <br />
+              <template v-if="!isNaN(quota.relative) && quota.definition !== 'none'">
+                <span v-text="personalStorageLabel" />
```

Comment:
They way I read this it can have `personalStorageLabel` followed by `personalStorageDetailsLabel` which provides the same information twice?